doc/conf.py

```diff
@@ -199,36 +197,81 @@ def __getattr__(cls, name):
 
 # Custom sidebar templates, must be a dictionary that maps document names
 # to template names.
-#
-# This is required for the alabaster theme
-# refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
 html_sidebars = {
     "**": [
-        "logo-text.html",
         "searchbox.html",
         "globaltoc.html",
     ]
 }
 
-
 # -- Xanadu theme ---------------------------------------------------------
-html_theme = "xanadu_theme"
-html_theme_path = ["."]
+html_theme = "xanadu"
 
 # xanadu theme options (see theme.conf for more information)
 html_theme_options = {
-    # Set the name of the project to appear in the left sidebar.
-    "project_nav_name": "PennyLane-Lightning",
-    # Path to a touch icon
-    "touch_icon": "logo_new.png",
-    "large_toc": True,
-    "navigation_button": "#19b37b",
-    "navigation_button_hover": "#0e714d",
-    "toc_caption": "#19b37b",
-    "toc_hover": "#19b37b",
-    "table_header_bg": "#edf7f4",
-    "table_header_border": "#19b37b",
-    "download_button": "#19b37b",
+    "navbar_wordmark_path": "_static/pl_wordmark.png",
+    # Specifying #19b37b is more correct but does not match the other PL websites.
```

Comment:
What does "more correct" mean?

Reply:
Ah, so the Xanadu logo currently shown in the navbar has the colour of the traditional Xanadu logo (i.e., teal) instead of PennyLane (i.e., green) which is inconsistent with the rest of our Sphinx websites.